routers/user_router.py

```diff
@@ -47,4 +48,15 @@ def delete_user(user_email: str, session: Session = Depends(get_db)) -> user_sch
     except ValueError as e:
         raise HTTPException(status_code=404, detail=str(e))
     return user_schema.UserDeletedResponse(message=f"user with email{user_email}, was successfully deleted")
+
+@router.put("/users/", status_code=status.HTTP_200_OK, response_model=user_schema.MentorAssignedResponse)
```

Comment:
I think this route is not necessary. 

If @Dharld finishes with the "update" endpoint, assigning a mentor is as easy as just updating the mentorEmail column

Reply:
I feel like this route is not necessary

Reply:
what do you suggest because I feel like it is

Reply:
Let's just remove this route and what we would do is that if we want to update somebody;s mentor, we just use the regular [PUT endpoint](https://github.com/PkFokam-Alumni-Network/CommunityBackendService/pull/33)